src/cryptonote_core/blockchain.cpp

```diff
@@ -363,8 +374,8 @@ bool Blockchain::init(BlockchainDB* db, const network_type nettype, bool offline
   }
   if (m_nettype == FAKECHAIN)
   {
-    for (size_t n = 0; test_options->hard_forks[n].first; ++n)
-      m_hardfork->add_fork(test_options->hard_forks[n].first, test_options->hard_forks[n].second, 0, n + 1);
+    for (size_t n = 0; n < sizeof(fakechain_hard_forks) / sizeof(fakechain_hard_forks[0]); ++n)
+      m_hardfork->add_fork(fakechain_hard_forks[n].version, fakechain_hard_forks[n].height, fakechain_hard_forks[n].threshold, fakechain_hard_forks[n].time);
```

Comment:
That is destroying the core tests. The fake chain has custom hard fork points for a reason.

Reply:
Yes, this patch seems unnecessary.

Reply:
I should've ran the core tests! Reverted this